lib/preprocess.js

```diff
@@ -51,14 +51,15 @@ export const preprocess = async ({
   roundIndex,
   fetchMeasurements,
   recordTelemetry,
-  logger
+  logger,
+  retries = 14
```

Comment:
Can we use a more descriptive property name please? `retries` does not indicate what will be retried.

```suggestion
  fetchMeasurementsRetries = 14
```

or perhaps this slightly shorter variant would be still ok

```suggestion
  fetchRetries = 14
```